### PR TITLE
Trailing return audit

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Linting would be too much of a pain

### CODE REVIEW

File: code/game/objects/machinery/computer/station_alert.dm [hunk 1]
```diff
@@ -102,5 +102,4 @@
 		icon_state = "atmos2"
 	else
 		icon_state = "atmos"
-	..()
```
Comment (SplinterGP) [inline code/game/objects/machinery/computer/station_alert.dm:105]: sounds like a woops

File: code/game/objects/machinery/computer/station_alert.dm [hunk 2]
```diff
@@ -102,5 +102,4 @@
...
-	return
+
```
Comment (SplinterGP) [inline code/game/objects/machinery/computer/station_alert.dm:105]: looks like a woops

File: code/modules/clothing/masks/miscellaneous.dm [hunk 3]
```diff
@@ -13,8 +13,8 @@
 	if (src == user.wear_mask)
 		return
 	else
-		..()
-	return
+		return ..()
```
Comment (SplinterGP) [inline code/modules/clothing/masks/miscellaneous.dm:16]:
```suggestion
	return ..()
```

File: code/modules/clothing/masks/miscellaneous.dm [hunk 4]
```diff
@@ -13,8 +13,8 @@
 	if (src == user.wear_mask)
 		return
 	else
```
Comment (SplinterGP) [inline code/modules/clothing/masks/miscellaneous.dm:15]:
```suggestion
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/computer/station_alert.dm (hunk 2 above)
Comment (TiviPlus) [reply code/game/objects/machinery/computer/station_alert.dm:105]: This is process() so I

File: code/game/objects/machinery/computer/station_alert.dm [hunk 5]
```diff
@@ -103,4 +103,4 @@
 	else
 		icon_state = "atmos"
 	..()
```
Comment (SplinterGP) [inline code/game/objects/machinery/computer/station_alert.dm:105]:
```suggestion
	return ..()
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/computer/station_alert.dm (hunk 5 above)
Comment (TiviPlus) [reply code/game/objects/machinery/computer/station_alert.dm:105]: process shouldnt return values unless its process kill

Comment (SplinterGP) [reply code/game/objects/machinery/computer/station_alert.dm:105]: should because some parent proc call might call it

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**